gke-batch-refarch/02_platform/monitoring/install-prometheus.sh

```diff
@@ -9,6 +9,7 @@ cd kube-prometheus && \
 kubectl apply --server-side -f manifests/setup && \
 kubectl wait \
     --for condition=Established \
+    --timeout=180s \
```

Comment:
Removing the timeout will cause it to use the default 30s, will that always be enough time?